lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
Also, I was thinking we add the changes to the content array [here](https://github.com/DoSomething/phoenix/blob/72e94a3639cff8c75cd167118975a59b591904b8/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc#L210) and not to the backend field